jenkins/asan-param.yml

```diff
@@ -164,12 +164,29 @@
         echo "${TRIGGERED_BUILD_NUMBERS_percona_server_8_0_pipeline}" > PIPELINE_BUILD_NUMBER
         gunzip build.log.gz
     publishers:
-    - warnings:
-        workspace-file-scanners:
-        - file-pattern: 'build.log'
-          scanner: 'GNU C Compiler 4 (gcc)'
-        run-always: true
-        shouldDetectModules: false
+    - raw:
```

Comment:
can we use `recordIssues enabledForFailure: true, tools: [gcc(pattern: 'build.log')]` here instead of raw xml?

Reply:
We can't, I tried already

Reply:
Sources does not contain following syntax: https://opendev.org/jjb/jenkins-job-builder/src/branch/master/jenkins_jobs/modules/publishers.py